examples/tasks/iostat-file.yml

```diff
@@ -0,0 +1,23 @@
+---
+version: 1
+schedule:
+  type: simple
+  interval: 2s
+workflow:
+  collect:
+    metrics:
+      "/intel/linux/iostat/avg-cpu/%user": {}
```

Comment:
Maybe we want to close this PR and reopen it with the pluginsync PR. Test are failing because `linux` got removed from the namespace.